OfficeAddin/05 Using modern JavaScript/Lab.md

```diff
@@ -1,51 +1,47 @@
-# Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
+# Build an Office Add-in using modern JavaScript tools and techniques
```

Comment:
@andrewconnell is the content duplicated in this file?

Reply:
The demos are excerpts of the labs... the updates requested in this review will be applied to the labs as well.